src/Ditto/ReplicatingConsumer.cs

```diff
@@ -43,6 +43,14 @@ public async Task ConsumeAsync(string eventType, ResolvedEvent resolvedEvent)
         {
             if (string.IsNullOrWhiteSpace(eventType)) throw new ArgumentException("Event type required", nameof(eventType));
 
+            if (_settings.EventNumberToStopAt.HasValue && resolvedEvent.OriginalEventNumber > _settings.EventNumberToStopAt.Value)
+            {
+                _logger.Error("Please stop ditto as it has reached the event number to stop at. Skipping processing." +
```

Comment:
You might want to add a space at the end 🙂

Reply:
Thanks